fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -537,66 +568,54 @@ public void testSearchPatientWithIdentifierOfType() {
         Bundle bundle = response.readEntity(Bundle.class);
```

Comment:
Would be good to update/add a test in SearchReferenceTest.java for case sensitivity/insensitivity as well.

Reply:
Done.